### PR TITLE
11.0 mig attachment base synchronize

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

File: attachment_base_synchronize/data/cron.xml [hunk 1]
```diff
@@ -8,8 +8,9 @@
             <field name="numbercall">-1</field>
             <field name="active">False</field>
             <field name="doall" eval="False" />
-            <field name="model">ir.attachment.metadata</field>
-            <field name="function">run_attachment_metadata_scheduler</field>
+            <field name="model_id" ref="model_ir_attachment_metadata"/>
+            <field name="state">code</field>
+            <field name="code">model.run_attachment_metadata_scheduler()</field>
```
Comment (thomaspaulb) [inline attachment_base_synchronize/data/cron.xml]: Shouldn't this be `model_id.run_....` ?

File: attachment_base_synchronize/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+
+Contributors
+------------
+
+* Valentin CHEMIERE <valentin.chemiere@akretion.com>
+* Florian da Costa <florian.dacosta@akretion.com>
+* Angel Moya <http://angelmoya.es>
```
Comment (thomaspaulb) [inline attachment_base_synchronize/README.rst:65]: Add yours

File: attachment_base_synchronize/data/cron.xml (hunk 1 above)
Comment (Kiplangatdan) [reply attachment_base_synchronize/data/cron.xml]: Nope, it's model.cron() [example](https://github.com/OCA/OCB/blob/11.0/addons/account_asset/data/account_asset_data.xml#L7)

**Review by thomaspaulb — COMMENTED** [review]
Approved.

**Pull request closed without merging**